packages/drafts-realm/.realm.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Demo Workspace",
+  "name": "Drafts Workspace",
   "backgroundURL": "https://ipfs.io/ipfs/QmX7uFUZubtjfnrnkn73VFFoGkyaoEAhKAi1t39UR2o179",
   "iconURL": "https://ipfs.io/ipfs/QmQJ4AYtLLvi1kiVWWM2n8zisJaeWCb53XHCywLSAiF5nP"
```

Comment:
Let's vary the backgroundURL and iconURL for drafts-realm so we can easily distinguish it from published-realm